src/TestFramework/TestFramework/Assertions/Assert.IsNull.cs

```diff
@@ -163,7 +145,7 @@ public static void IsNull(object? value, [InterpolatedStringHandlerArgument(name
     /// <exception cref="AssertFailedException">
     /// Thrown if <paramref name="value"/> is not null.
     /// </exception>
-    public static void IsNull(object? value, string? message)
+    public static void IsNull(object? value, [CallerArgumentExpression(nameof(value))] string message = "")
```

Comment:
We can discuss about the design, what I had in mind was to still allow the user to set a custom message but for us to use the capture mechanism to enrich the default message (always displayed - or with some verbosity setting in case).

Reply:
I don't know if we should have an analyzer to ensure people don't set this value or if we should use the trick from @sharwell (bait and switch).